src/models/NewsCardType.tsx

```diff
@@ -0,0 +1,7 @@
+export type NewsCardType = {
```

Comment:
please do no use `type` postfix in models.

models files should start from lowercase. So it gonna be `src/models/news`.

Also `Card` postfix not a good name for model. Model name it's an abstraction. Like `Article`, `News` and etc. So I would name it `export type News = ...`